.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Core Dataset",
+    "image": "mcr.microsoft.com/devcontainers/python:3.11",
+    "features": {
+        "ghcr.io/stuartleeks/dev-container-features/shell-history:0": {}
+    }
+}
```

Comment:
With this, you can run the pipeline in your browser using [GitHub Codespaces](https://github.com/codespaces).

A simple way to make future contributors life easier as getting a development environment is one click away.